### ISSUE TITLE
Dependents | Fix net worth accordion header

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#107964]
## Description

While reviewing the dependents management forms, I noticed that the accordion on the net worth page has an `H2` making the same level as the progress bar, it should be an `H4` to fit within the page content

<img width="800" alt="page title is an h3, accordion under it has an h2 when it should be an h4" src="https://github.com/user-attachments/assets/31370f5e-eace-47c1-b09b-37b19cbaeccd" />

## Tasks

- [ ] Update accordion header to `H4`
- [ ] Add tests to prevent regression

## Acceptance Criteria

- All tasks complete
- All tests passing

**Comment by TaiWilkin** [comment]
@Mottie Are these the desired heading levels?
<img width="1204" alt="Image" src="https://github.com/user-attachments/assets/dbd94d02-fb6c-44d5-bcf7-9564897df16d" />
<img width="1196" alt="Image" src="https://github.com/user-attachments/assets/647860f4-a186-4dc8-ba5b-48e1a9f2dd42" />

**Comment by Mottie** [comment]
Oops, on the upload page, those last two headers should be `H4`s instead of `H3`s

**Comment by TaiWilkin** [comment]
> Oops, on the upload page, those last two headers should be `H4`s instead of `H3`s

@Mottie "Supporting evidence you need to submit" *and* "Submit your files online"?

**Comment by Mottie** [comment]
Yes, both should be h4s because they are under the page header "Upload your supporting evidence to add your child" (h3)
